digital-twin/e2e-tests/digital-twin-e2e-tests-common/src/main/java/com/microsoft/azure/sdk/iot/digitaltwin/e2e/tests/DigitalTwinServiceClientE2ETests.java

```diff
@@ -58,14 +59,19 @@ public void setUpTest() throws IotHubException, IOException, URISyntaxException
     }
 
     @Test
+    // TODO
+    @Ignore("404 response from server")
```

Comment:
Our nightly run shows that this test passed, we should not ignore it.

Reply:
sounds good.